src/lib/y2storage/callbacks/probe.rb

```diff
@@ -19,11 +19,116 @@
 
 require "y2storage/callbacks/libstorage_callback"
 
+Yast.import "Mode"
+Yast.import "Label"
+Yast.import "Popup"
+
 module Y2Storage
   module Callbacks
     # Class to implement callbacks used during libstorage-ng probe
-    class Probe < Storage::ProbeCallbacks
+    class Probe < Storage::ProbeCallbacksV3
       include LibstorageCallback
+
+      # Callback for missing commands during probing.
+      #
+      # @param message [String] error title coaming from libstorage-ng
+      #   (in the ASCII-8BIT encoding! see https://sourceforge.net/p/swig/feature-requests/89/)
+      # @param what [String] details coming from libstorage-ng (in the ASCII-8BIT encoding!)
+      # @param command [String] missing command coming from libstorage-ng (in the ASCII-8BIT encoding!)
+      # @param used_features used features bit field coming from libstorage-ng
```

Comment:
used features type missing. Is it integer or bitmap?

Reply:
I mean ideally we should specify its ruby type. I expect it is `Integer` right?

Reply:
Indeed, the type is missing. Why does yardoc not compain?